frontend/src/views/Systems.vue

```diff
@@ -280,11 +227,51 @@ export default Vue.extend({
     fahrenheit() {
       window.localStorage.setItem("units", this.fahrenheit ? "F" : "C");
     }
+  },
+  computed: {
+    // search for node modules and return the result
+    resultQuery() {
+      if (this.searchQuery) {
+        return this.systems.filter(item => {
+          return this.searchQuery
+            .toLowerCase()
+            .split(" ")
+            .every(v => item.Identifier.toLowerCase().includes(v));
+        });
+      } else {
+        return this.systems;
+      }
```

Comment:
Rewrote slightly to use types and `$data` to prevent Vetur errors.

```suggestion
      if (!this.searchQuery) {
        return this.$data.systems;
      }

      return this.$data.systems.filter((item: GardenSystem) => {
        return this.searchQuery
          .toLowerCase()
          .split(" ")
          .every(v => item.Identifier.toLowerCase().includes(v));
      });
```